configs/_base_/models/swin/swin_large.py

```diff
@@ -0,0 +1,8 @@
+_base_ = 'swin_tiny.py'
+model = dict(
```

Comment:
Is it a good idea to add multiple configs files to `_base_/models`? They can be added to each  config file at `recognition/swin` actually